pilot/pkg/features/pilot.go

```diff
@@ -673,6 +673,9 @@ var (
 	// User should not rely on builtin resource labels, this flag will be removed in future releases(1.20).
 	EnableOTELBuiltinResourceLables = env.Register("ENABLE_OTEL_BUILTIN_RESOURCE_LABELS", false,
 		"If enabled, envoy will send builtin lables(e.g. node_name) via OTel sink.").Get()
+
+	// By default, Istio assumes CA credential name to have the format <credential>-cacert, use TLS_CUSTOM_CA_CREDENTIAL to override.
+	TLSCustomCaCredential = env.Register("TLS_CUSTOM_CA_CREDENTIAL", "", "The Custom CA credential name used in Client/Server TLS Settings").Get()
```

Comment:
This is a hack way, and i think this cannot work:  how can you distinguish different  cacerts

Reply:
https://github.com/kubernetes/enhancements/issues/3257 is probably the "proper" way to do that although its a bit early